packet/client.go

```diff
@@ -14,17 +14,42 @@ import (
 	"github.com/pkg/errors"
 	"github.com/tinkerbell/boots/httplog"
 	tinkClient "github.com/tinkerbell/tink/client"
+	tw "github.com/tinkerbell/tink/protos/workflow"
 )
 
 type hardwareGetter interface {
 }
 
+type workflowGetter interface {
+	getWorkflowConnection() tw.WorkflowSvcClient
+	setWorkflowConnection() error
+}
+
+type workflow struct {
+	wsc tw.WorkflowSvcClient
+}
+
+func (w *workflow) setWorkflowConnection() error {
+	wsc, err := tinkClient.TinkWorkflowClient()
+	if err != nil {
+		return err
+	}
+	(*w).wsc = wsc
+	return nil
+}
+
+func (w *workflow) getWorkflowConnection() tw.WorkflowSvcClient {
+	return w.wsc
+}
```

Comment:
we no longer need any of this right? Now that we are just passing the wcl into `isHardwareHavingActiveWorkflow` and we don't have different concrete types. Better to get rid of all this unneeded code then.

Reply:
It is not related `isHardwareHavingActiveWorkflow`. You may want to check recent updates in endpoints.go. 

This method passes the tink workflow client to `GetWorkflowsFromTink` in endpoints.go.

Reply:
As discussed, I have kept `getWorkflowConeection` and have removed `setWorkflowConnection`